tap_trello/streams.py

```diff
@@ -212,13 +217,32 @@ def get_records(self, format_values, additional_params=None):
             )
 
         for rec in records:
-            yield self.modify_record(rec, parent_id_list = format_values)
+            yield self.modify_record(rec, parent_id_list = format_values, custom_fields_map = custom_fields_map)
 
 
     def sync(self):
         for rec in self.get_records(self.get_format_values()):
             yield rec
 
+class AddCustomFields(Mixin):
+    def build_custom_fields_map(self, **kwargs):
+        custom_fields_map = {}
+        board_id_list = kwargs['parent_id_list']
+        assert len(board_id_list) == 1
+        custom_fields = self.client.get('/boards/{}/customFields'.format(board_id_list[0])) # pylint: disable=no-member
```

Comment:
Shouldn't this be using the set of all field names/ids for every board?

Reply:
I think I see it, a docstring would be good here if I'm reading correctly, that it's called in `get_records` of the parent and built per-parent.